### DISCUSSION TITLE
other players spectate the host's song select in multiplayer

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#19008] [Ideas] [answered]
it's a setting that can be checked on or off by the room creator
other players in a regular multiplayer match will be able to see what beatmaps the host has(or rather, the beatmaps that are visible in the carousel of the spectated player) 

this will allow players to see new beatmaps that the host has that they've never seen and make suggestions on what to play

players can also individually turn this setting on, off, or limit the feature to certain people in the settings if they don't want others to see what beatmaps they have

in cases where a unsubmitted beatmap shows up, it will be as if the beatmap didn't exist at all on the carousel(but the host can still pick it)

**Comment by peppy** [comment, marked as answer]
Please use the queue mode which allows players to add beatmaps instead. This sounds complicated and less useful than what we already have implemented.

**Reply by iSimpFor** [reply]
hmmm

even though this idea is mostly redundant, i still like the idea of being able to spectate other people's song select since they're likely to find new beatmaps that they may want to check out

maybe this idea can be put in in another way?

**Reply by peppy** [reply]
Use the beatmap listing
